client/rest-high-level/src/test/java/org/elasticsearch/client/SecurityRequestConvertersTests.java

```diff
@@ -101,6 +104,25 @@ public void testPutRoleMapping() throws IOException {
         assertToXContentBody(putRoleMappingRequest, request.getEntity());
     }
 
+    public void testGetRoleMappings() throws IOException {
+        int noOfRoleMappingNames = randomIntBetween(0, 2);
+        final String[] roleMappingNames = randomArray(noOfRoleMappingNames, noOfRoleMappingNames, String[]::new, () -> randomAlphaOfLength(
+                5));
+        final GetRoleMappingsRequest getRoleMappingsRequest = new GetRoleMappingsRequest(roleMappingNames);
+
+        final Request request = SecurityRequestConverters.getRoleMappings(getRoleMappingsRequest);
+
+        assertEquals(HttpGet.METHOD_NAME, request.getMethod());
+        if (noOfRoleMappingNames == 0) {
+            assertEquals("/_xpack/security/role_mapping", request.getEndpoint());
+        } else {
+            assertEquals("/_xpack/security/role_mapping/" + Strings.collectionToCommaDelimitedString(getRoleMappingsRequest
```

Comment:
I don't like splitting a statement across multiple lines like this. I'd prefer to have the whole `Strings.collection...` call on one line.